pkg/client/tx/client.go

```diff
@@ -365,34 +365,6 @@ func (tClient *txClient) goSubscribeToOwnTxs(ctx context.Context) {
 		txHashHex := txHashBytesToNormalizedHex(comettypes.Tx(txResult.Tx).Hash())
 
 		tClient.txsMutex.Lock()
```

Comment:
NIT: Wdyt of renaming `tClient` to `txClient`?

I was confused what it was referring to at first.